cognite/client/_api/entity_matching.py

```diff
@@ -0,0 +1,220 @@
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+from requests import Response
+
+from cognite.client import utils
+from cognite.client._api_client import APIClient
+from cognite.client.data_classes._base import CogniteResource
+from cognite.client.data_classes.contextualization import (
+    ContextualizationJob,
+    EntityMatchingModel,
+    EntityMatchingModelList,
+    EntityMatchingModelUpdate,
+)
+from cognite.client.utils._auxiliary import convert_true_match
+
+
+class EntityMatchingAPI(APIClient):
+    _RESOURCE_PATH = EntityMatchingModel._RESOURCE_PATH
+    _LIST_CLASS = EntityMatchingModelList
+
```

Comment:
no __call__ method?

Reply:
we don't have cursors or pagination, so this whole chunking seems not needed